static/js/app.js

```diff
@@ -847,6 +870,95 @@ $(function () {
             this.fetchMarkers();
             this.updateFilterString();
         },
+        loadFilterFromParameters: function() {
+            var bool_atrs = {};
+            bool_atrs["show_discussions"] = this.showDiscussion;
```

Comment:
Yes, I mean

    bool_attrs = ["show_markers", ...];
    $.each(bool_attrs, function(attr) {
        $(attr).prop("checked", this[attr] == '1');
    });